### ISSUE TITLE
Zim files created in dev should default to manual refresh

### ISSUE THREAD

**Issue by Popolechien** [openzim/zimfarm#1009]
The typical manner to create a new recipe is to copy an exisiting one, change the parameters and assign it to dev. It happens quite often that one forgets to change the update settings and leave it to `monthly`, meaning that if the recipe is not successful and not moved to PROD it may keep on generating new zim files every month, potentially clogging the dev repo (by "not successful" I understand that a zim is produced, but does not meet quality requirements and thus could be considered junk).

Now the typical fix for that would be to say "check your parameters", but I think having a harder check, _i.e._ the `periodicity`parameter automatically change to `manual` as soon as the warehouse path is set to /.hidden/dev would be much more efficient.

**Comment by benoit74** [comment]
I agree that something needs to be done, but I'm not sure the solution proposed is adequate.

If one forgets to set the `periodicity` to `manual` when in dev and the software is modified to do this change automatically, then I'm pretty sure that the `periodicity` will stay to `manual` once move to prod, so we will only have move the problem.

Sorry that this comment does not propose an alternative.

**Comment by benoit74** [comment]
Maybe implementing warnings in the UI would help:
- warn when a recipe is configured in dev and periodicity is not set to manual
- warn when a recipe is configured in prod and periodicity is manual

I can see many other use cases for such warnings.

**Comment by benoit74** [comment]
With the move to the CMS, we will soon not know anymore from Zimfarm side if a given recipe is in prod or in dev.

And I feel like the rest is more for the CMS:
- CMS will warn when a ZIM has not been updated for too long
- CMS will display any new ZIM arriving in staging, so it will be so annoying if someone forgets to set the recipe to "manual" that we will fix it

Not sure we have much left to implement here.
